src/main/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs.java

```diff
@@ -171,6 +168,16 @@ public final class GenotypeGVCFs extends VariantLocusWalker {
     @ArgumentCollection
     private GenomicsDBArgumentCollection genomicsdbArgs = new GenomicsDBArgumentCollection();
 
+    /**
+     * This option can only be activated if intervals are specified.
+     */
+    @Deprecated
+    @Argument(fullName= ONLY_OUTPUT_CALLS_STARTING_IN_INTERVALS_FULL_NAME,
```

Comment:
I think this was previously advanced as well, I'd tag it again.

Reply:
Also, I believe we should be using barclay's `@DeprecatedFeature` instead of java deprecated.  You can include a "detail" message in it that explains what to use instead.